unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -29,8 +29,8 @@ public class AvatarEditorHUDController : IHUD
     private const string THIRD_PARTY_COLLECTIONS_FEATURE_FLAG = "third_party_collections";
     private const int REQUESTS_COOLDOWN_TIME = 60;
     internal const string EQUIP_WEARABLE_METRIC = "equip_wearable";
-    protected static readonly string[] categoriesThatMustHaveSelection = { Categories.BODY_SHAPE, Categories.UPPER_BODY, Categories.LOWER_BODY, Categories.FEET, Categories.EYES, Categories.EYEBROWS, Categories.MOUTH };
-    protected static readonly string[] categoriesToRandomize = { Categories.HAIR, Categories.EYES, Categories.EYEBROWS, Categories.MOUTH, Categories.FACIAL, Categories.HAIR, Categories.UPPER_BODY, Categories.LOWER_BODY, Categories.FEET };
+    protected static readonly string[] categoriesThatMustHaveSelection = { WearableLiterals.Categories.BODY_SHAPE, WearableLiterals.Categories.UPPER_BODY, WearableLiterals.Categories.LOWER_BODY, WearableLiterals.Categories.FEET, WearableLiterals.Categories.EYES, WearableLiterals.Categories.EYEBROWS, WearableLiterals.Categories.MOUTH };
```

Comment:
i think you can avoid `WearableLiterals.` because at top is set `using Categories = WearableLiterals.Categories;`

Reply:
Done: https://github.com/decentraland/unity-renderer/pull/4933/commits/0a2f046e478f6cf4cd09206b60ab8863940332f4